Makefile

```diff
@@ -212,6 +220,10 @@ haskell_main_file      := driver.md
 haskell_main_filename  := $(basename $(notdir $(haskell_main_file)))
 haskell_kompiled       := $(haskell_dir)/$(haskell_main_filename)-kompiled/definition.kore
 
+ifeq ($(UNAME_S),Darwin)
+$(KEVM_LIB)/$(haskell_kompiled): $(libsecp256k1_out)
+endif
```

Comment:
```suggestion
ifeq ($(UNAME_S),Darwin)
$(KEVM_LIB)/$(haskell_kompiled): $(libsecp256k1_out)
$(KEVM_LIB)/$(llvm_kompiled): $(libsecp256k1_out)
endif
```

Reply:
I can build without using both, but it does no harm to build both.